src/relstorage/_util.py

```diff
@@ -604,3 +608,12 @@ def __repr__(self):
             self.needs_fetchall,
             self.last_stmt,
         )
+
+
+
+try:
+    hashed = md5_original('test'.encode())
+    md5 = md5_original
+except ValueError:
+    md5 = partial(md5_original, usedforsecurity=False)
```

Comment:
We seem to be getting this branch everywhere, because all the tests are failing with `TypeError: openssl_md5() takes at most 1 argument (2 given)`.

Reply:
I believe this was because of an invalid line I accidentally left in